docs/.vuepress/config.js

```diff
@@ -31,6 +31,64 @@ const dirTree = require('directory-tree')
 const path = require("path");
 const fs = require("fs");
 const versions = require("./versions.js");
+const installMethods = [
```

Comment:
Is this all related to broken links checker not exiting correctly?

Reply:
Nope it's just cleanup from a weirdly placed file